README.md

```diff
@@ -4,6 +4,8 @@
 # PreREISE
 This package gathers and builds demand, hydro, solar, and wind profiles. The
 profiles are needed to run scenarios on the U.S. electrical grid.
+This package is part of a set of packages representing Breakthrough Energy's energy system model.
+More information regarding the installation of the whole model as well as the contribution guide can be found [here](https://breakthrough-energy.github.io/docs/).
```

Comment:
I would write:
`Detailed installation and contribution guides can be found [here](https://breakthrough-energy.github.io/docs/)`

Reply:
I left that part mostly unchanged.